src/exception/repo/program.ts

```diff
@@ -35,10 +35,9 @@ const programExceptionSchema = new mongoose.Schema<ProgramException>({
   ],
 });
 
-const ProgramExceptionModel = mongoose.model<ProgramException>(
-  'ProgramException',
-  programExceptionSchema,
-);
+const ProgramExceptionModel =
+  mongoose.models.ProgramException ||
+  mongoose.model<ProgramException>('ProgramException', programExceptionSchema);
```

Comment:
Not 100% sure what the difference is here. Is this a Mongoose nuance?

Reply:
this is a fun one. 
live watching for tests to re-run causes mongoose to try and reinitialize models, which it shouldn't do so it throws an error. this just makes sure these models haven't been init already, so no testing errors

Reply:
Sounds like they have a bug or something. Maybe a comment here would be useful to explain intent?